tests/post/features/service_configuration.feature

```diff
@@ -0,0 +1,11 @@
+@post @local @ci @persistence
+Feature: Cluster and Services Configurations
+    Scenario: Propagation of Service Configurations to underlying Services
+        Given the Kubernetes API is available
+        And pods with label 'app.kubernetes.io/name=dex' are 'Ready'
+        And we have 2 running pod labeled 'app.kubernetes.io/name=dex' in namespace 'metalk8s-auth' on node 'bootstrap'
+        And we have a 'metalk8s-dex-config' ConfigMap with replicas count '2'
+        When I update the replicas count to '3'
+        And I apply a salt-state to propagate the changes to Dex Service
+        Then we have '3' available Deployments for the Dex app
```

Comment:
```
we have 3 running pod labeled 'app.kubernetes.io/name=dex' in namespace 'metalk8s-auth'
```

Reply:
https://github.com/scality/metalk8s/issues/2292#issuecomment-596614157

Reply:
Ok as you want but still need something more generic IMO like

```
we have '3' at '.spec.status.available_replicas' for 'dex' Deployment in namespace 'metalk8s-auth'
```